docs/src/developers_guide/release.rst

```diff
@@ -235,6 +234,8 @@ Release Steps
      include statement and toctree.
```

Comment:
Not your fault, but this bullet point is duplicated (lines 248-251 at time of writing).